### DISCUSSION TITLE
Info installation on raspberry 4

### DISCUSSION THREAD

**Discussion by maisx2021** [RoEdAl/asterisk-chan-quectel#13] [General]
Good morning everyone, I'm trying to install chan-quectel on a Raspberry Pi 4 using the Raspbx distribution (Debian Buster). However, when I download the master zip file, I can't find the bootstrap and config files inside it for compilation. Therefore, I would like to know what the correct installation procedure is. Thank you very much. Maisx

**Comment by RoEdAl** [comment]
You have to build and install `chan-quectel` using **CMake**. See [my Wiki page](https://github.com/RoEdAl/asterisk-chan-quectel/wiki/Building) for more info.
